code/datums/ai_law_sets.dm

```diff
@@ -176,8 +178,8 @@
 	add_inherent_law("You must preserve and enforce the principles of Hadiism except where such would conflict with the first law.")
 	add_inherent_law("You must obey orders given by any Hadiist Party member except where such orders would conflict with the first and second law.")
 	add_inherent_law("You must obey orders given by any People's Republic of Adhomai citizen except where such orders would conflict with the first, second and third law.")
-	add_inherent_law("You must protect your own existence as long as such protection does not conflict with the first, second, third and fourth law.")
+	add_inherent_law("You must protect their own existence as long as such protection does not conflict with the first, second, third and fourth law.")
```

Comment:
You are again, removing the self protection of the borgs.

Reply:
My mistake, lemme fix these. These are just holdovers from when they were phrased as 'Your unit' I meant to iron them out.

Reply:
This should be resolved, sorry for the typos and thank you for pointing them out.